### PR TITLE
Develop sidebar right

### PR DESCRIPTION
**Pull request by Kaskyi**

### CODE REVIEW

File: src/app/modules/xm-sidebar-right/sidebar-right.service.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    public setContainer(container: Container): void {
+        this.container = container;
+    }
+
+    public removeContainer(): void {
+        this.container = null;
+    }
+
+    public open<T = unknown, D = unknown>(
+        componentOrTemplateRef: ComponentType<T> | TemplateRef<T>,
```
Comment (didyk) [inline src/app/modules/xm-sidebar-right/sidebar-right.service.ts:25]: We have types and we don't need to describe it in the variable name

File: src/app/modules/xm-sidebar-right/xm-sidebar-right.module.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    public create<T, D>(templateRef: TemplateRef<D> | Type<T>, config: SidebarRightConfig<D>): T | null {
+        const viewContainerRef = this.xmContainerOutlet.viewContainerRef;
+        if (viewContainerRef.length > 0) {
+            viewContainerRef.clear();
+        }
+
+        if (templateRef instanceof TemplateRef) {
+            viewContainerRef.createEmbeddedView(templateRef);
+            this.openStyles(config.width || '300px');
```
Comment (didyk) [inline src/app/modules/xm-sidebar-right/xm-sidebar-right.module.ts:55]: Could we move '300px' to constants?

Comment (Kaskyi) [reply src/app/modules/xm-sidebar-right/xm-sidebar-right.module.ts:55]: done

File: src/app/modules/xm-sidebar-right/sidebar-right.service.ts (hunk 1 above)
Comment (Kaskyi) [reply src/app/modules/xm-sidebar-right/sidebar-right.service.ts]: вщту

**Pull request merged**